BUILD_COMMANDLINE.md

```diff
@@ -57,76 +57,96 @@
 
-    Optional: If you wish, you can use a Python virtual environment to manage your dependencies. Before installing
-    Verovio, create and activate a virtual environment. 
-    
-    ```shell
-    python3 -m venv ./.venv
-    source ./.venv/bin/activate
-    ```
+     Optional: If you wish, you can use a Python virtual environment to manage your dependencies. Before installing Verovio, create and activate a virtual environment. 
     
+     ```shell
+     python3 -m venv ./.venv
+     source ./.venv/bin/activate
+     ```
+ 
     This will install your Python libraries in the local `.venv` directory. Once your virtual environment is active you can continue to installing Verovio.
 
     To build the images with Verovio, you need Python3 to be installed with the `verovio` module. This can be installed with:
 
     ```shell
     pip install verovio
     ```
-    
-2. Initialize the build process
+
+3. Build MEI artifacts
 
    * Switch to your clone’s directory:
 
      ```shell
      cd [YOUR-CLONE-LOCATION]
      ```
 
-   * Call the Apache Ant™ init task:
+   * Call an Apache Ant™ task
+     
+     For building any MEI artifacts you generally call Apache Ant™ by typing `ant` followed by a space and the name of the desired target:
 
      ```shell
-     ant init
+     ant [TASKNAME]
      ```
 
-3. Run the build process
+     * Build guidelines HTML:
 
-   * Build guidelines HTML:
+       ```shell
+       ant build-guidelines-html
+       ```
 
-     ```shell
-     ant -lib lib/saxon/saxon-he-11.5.jar build-guidelines-html
-     ```
+       The results of this build can be found in the web folder (`music-encoding/dist/guidelines/web`). The guidelines are stored in the `index.html` file.
 
-     The results of this build can be found in the web folder (`music-encoding/dist/guidelines/web`). The guidelines are stored in the `index.html` file.
+     * Build the RNG schema of a specific customization:
 
-     To generate the example images with Verovio, you need to run:
-     ```shell
-     ant generate-images-py
-     ```
+       ```shell
+       ant -Dcustomization.path="[/ABSOLUTE/PATH/TO/YOUR/CUSTOMIZATION]" build-rng
+       ```
 
-     **Note:** If you have installed your dependencies in a virtual environment, be sure to activate it prior to calling the Ant task. Activate it using:
-     
-     ```shell
-     source ./.venv/bin/activate
-     ```     
+     * Build the compiled ODD of a specific customization:
 
-   * Build the RNG schema of a specific customization:
+       ```shell
+       ant -Dcustomization.path="[/ABSOLUTE/PATH/TO/YOUR/CUSTOMIZATION]" build-compiled-odd
+       ```
+     * Build the RNG schema of a specific customization:
 
-     ```shell
-     ant -lib lib/saxon/saxon-he-11.5.jar -Dcustomization.path="[/ABSOLUTE/PATH/TO/YOUR/CUSTOMIZATION]" build-rng
-     ```
+       ```shell
+       ant -lib lib/saxon/saxon-he-11.5.jar -Dcustomization.path="[/ABSOLUTE/PATH/TO/YOUR/CUSTOMIZATION]" build-rng
+       ```
```

Comment:
Duplicated step from above? Also, this one here seems to be outdated since lib call is not needed anymore, right?

Reply:
fixed